articles/users/index.md

```diff
@@ -106,7 +106,15 @@ description: Learn about working with users in Auth0
   <li>
     <i class="icon icon-budicon-715"></i><a href="/users/search">User Search</a>
     <p>
-        Learn the best practices to use when retrieving users with API endpoints.
+        Learn how to search for users using the Management API.
     </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/users/search/query-syntax">Learn how you can use the query string syntax to build custom queries</a>
```

Comment:
is `/users/search/query-syntax` a valid url?

Reply:
Yes it's valid, it will navigate to the latest version by default (`/users/search/v3/query-syntax`) and the user can use the version dropdown to go other versions (v2) if they want. 

Only the sidebar needs the specific version as part of the URL.